docs/backend-api/resources/tracking/tracker/rules/rule_types.md

```diff
@@ -344,403 +379,561 @@ Autocontrol related tracked rules.
 
 #### parameters
 
-| name | description                           | type                                              |
-|------|---------------------------------------|---------------------------------------------------|
-| type | Default `type`: "check_engine_light". | [enum](../../../../getting-started.md#data-types) |
+| name           | description                                                          | type                                              |
+|----------------|----------------------------------------------------------------------|---------------------------------------------------|
+| type           | Default `type`: "strap_bolt".                                        | [enum](../../../../getting-started.md#data-types) |
+| primary_text   | Primary text of rule notification when padlock has been forced.      | string                                            |
+| secondary_text | Secondary text of rule notification when padlock has been installed. | string                                            |
+
+***
+
+### Tracker detach from the objects
+
+A rule that triggers when someone detach tracker from the object. Based on the configs on the device side.
+
+#### parameters
+
+| name | description               | type                                              |
+|------|---------------------------|---------------------------------------------------|
+| type | Default `type`: "detach". | [enum](../../../../getting-started.md#data-types) |
+
+***
+
+### Tracker switch ON/OFF
+
+A rule that triggers on tracker switch ON/OFF. Based on the configs on the device side.
+
+#### parameters
+
+| name           | description                                                   | type                                              |
+|----------------|---------------------------------------------------------------|---------------------------------------------------|
+| type           | Default `type`: "on_off".                                     | [enum](../../../../getting-started.md#data-types) |
+| primary_text   | Primary text of rule notification when tracker switched off.  | string                                            |
+| secondary_text | Secondary text of rule notification when tracker switched on. | string                                            |
+
+***
+
+### Tracker switched OFF or lost connection
+
+A rule that triggers when tracker loses connection with the server - gets red offline status and keeps it for X minutes.
+
+#### parameters
+
+| name           | description                                                                          | type                                              |
+|----------------|--------------------------------------------------------------------------------------|---------------------------------------------------|
+| type           | Default `type`: "offline".                                                           | [enum](../../../../getting-started.md#data-types) |
```

Comment:
>Default `type`: "offline".

It's not default. It's a mandatory, only one possible value. Relates to all rules.

Reply:
Fixed